lms/resources/_js_config.py

```diff
@@ -458,6 +460,14 @@ def _hypothesis_client(self):
             ]
         }
 
+    def set_groups(self, groups: List[HGroup]):
+        self._hypothesis_client["services"][0]["groups"] = [
+            g.groupid(self._authority) for g in groups
+        ]
+
+    def yolo_group_categories(self, group_categories):
```

Comment:
I endup sending the actual list of group categories instead of the URL for an API call is it was easier for me to setup.